test/index.test.js

```diff
@@ -13,11 +13,13 @@ const testPayloadPushTag = require('./data/github.push.tag.json');
 const testPayloadRelease = require('./data/github.release.json');
 const testPayloadReleaseBadAction = require('./data/github.release.badAction.json');
 const testPayloadTag = require('./data/github.tag.json');
-const testPayloadPushBadHead = require('./data/github.push.badHead.json');
```

Comment:
Can we also remove `./data/github.push.badHead.json` file?

Reply:
I removed it.